server/server.go

```diff
@@ -286,6 +293,18 @@ func (h RequestHandler) renderGenericMessage(res http.ResponseWriter, req *http.
 	return
 }
 
+func (h RequestHandler) renderGeneric401(res http.ResponseWriter, req *http.Request) {
+	res.WriteHeader(401)
+	title := h.translator.Translate("ErrGeneric401")
+	data := GenericMessageData{
+		Title:   title,
+		Message: h.translator.Translate("ErrGeneric401Message"),
+	}
+	h.renderGenericMessage(res, req, data)
+	return
+
+}
+
```

Comment:
this is only used in one place and it's very specific, inline it instead? if the pattern repeats of needing to handle multiple error codes generically then this pattern could be reused by also accepting a status code

Reply:
ie, make it `fmt.Sprintf(ErrGeneric%d)`?

Reply:
inlined it for now!